lib/vsc/administration/vo.py

```diff
@@ -220,76 +229,88 @@ def _create_fileset(self, filesystem_name, path, parent_fileset=None, fileset_na
         else:
             logging.info("Fileset %s already exists for VO %s ... not creating again.",
                          fileset_name, self.vo.vsc_id)
 
-        self.gpfs.chmod(0o770, path)
+        fs_backend.chmod(0o770, path)
 
         try:
             moderator = mkVscAccount(self.rest_client.account[self.vo.moderators[0]].get()[1])
         except HTTPError:
             logging.exception("Cannot obtain moderator information from account page, setting ownership to nobody")
-            self.gpfs.chown(pwd.getpwnam('nobody').pw_uid, fileset_group_owner_id, path)
+            fs_backend.chown(pwd.getpwnam('nobody').pw_uid, fileset_group_owner_id, path)
         except IndexError:
             logging.error("There is no moderator available for VO %s", self.vo.vsc_id)
-            self.gpfs.chown(pwd.getpwnam('nobody').pw_uid, fileset_group_owner_id, path)
+            fs_backend.chown(pwd.getpwnam('nobody').pw_uid, fileset_group_owner_id, path)
         else:
-            self.gpfs.chown(moderator.vsc_id_number, fileset_group_owner_id, path)
+            fs_backend.chown(moderator.vsc_id_number, fileset_group_owner_id, path)
 
     def create_data_fileset(self):
         """Create the VO's directory on the HPC data filesystem. Always set the quota."""
         path = self._data_path()
         try:
-            fs = self.storage[self.host_institute][VSC_DATA].filesystem
-        except AttributeError:
-            logging.exception("Trying to access non-existent attribute 'filesystem' in the data storage instance")
+            storage = self.storage[self.host_institute][VSC_DATA]
         except KeyError:
-            logging.exception("Trying to access non-existent field %s in the data storage dictionary", VSC_DATA)
-        self._create_fileset(fs, path)
+            errmsg = "Trying to access non-existent field %s in the data storage dictionary"
+            logging.exception(errmsg, VSC_DATA)
+        else:
+            self._create_fileset(storage, path)
 
     def create_data_shared_fileset(self):
         """Create a VO directory for sharing data on the HPC data filesystem. Always set the quota."""
         path = self._data_shared_path()
-        msg = "Trying to access non-existent"
         try:
-            fs = self.storage[self.host_institute][VSC_DATA_SHARED].filesystem
-        except AttributeError:
-            logging.exception("%s attribute 'filesystem' in the shared data storage instance", msg)
+            storage = self.storage[self.host_institute][VSC_DATA_SHARED]
         except KeyError:
-            logging.exception("%s field %s in the shared data storage dictionary", msg, VSC_DATA_SHARED)
-        self._create_fileset(fs, path,
-                             fileset_name=self.sharing_group.vsc_id,
-                             group_owner_id=self.sharing_group.vsc_id_number)
+            errmsg = "Trying to access non-existent field %s in the shared data storage dictionary"
```

Comment:
more Trying -> failed

Reply:
done in https://github.com/hpcugent/vsc-administration/pull/134/commits/d9d2f11e38cfec377807ac56339b0498d1b1400d